core/src/test/java/com/linecorp/armeria/server/Http1ServerDelayedCloseConnectionTest.java

```diff
@@ -50,6 +50,7 @@ protected void configure(ServerBuilder sb) {
                 return HttpResponse.builder()
                                    .ok()
                                    .content("OK\n")
+                                    .header(HttpHeaderNames.CONTENT_TYPE, "text/plain; charset=utf-8")
```

Comment:
I don't think we can simply alter the behavior since it can result in breaking changes like seen in the tests.

Rather, I think a more reasonable approach would be:
1) Remember if a `String` content has been set for a request/response
2) At build time, if a content-type hasn't been set, set `text/plain` as the content-type